libs/icu/Makefile

```diff
@@ -41,23 +41,36 @@ define Package/icu
   DEPENDS:=+libstdcpp +libpthread
 endef
 
+CONFIGURE_ARGS_ICU := \
+	--disable-debug \
+	--enable-release \
+	--enable-shared \
+	--enable-static \
+	--enable-draft \
+	--enable-renaming \
+	--disable-tracing \
+	--disable-extras \
+	--enable-dyload \
+	--enable-layout \
+	--enable-layoutex \
+				
+CONFIGURE_ARGS += $(CONFIGURE_ARGS_ICU) \
+	--disable-tools \
+	--disable-tests \
+	--disable-samples \
+	--with-cross-build="$(HOST_BUILD_DIR)"
+	
+HOST_CONFIGURE_ARGS += $(CONFIGURE_ARGS_ICU)
+
 define Build/Configure
```

Comment:
I have made several attempts at using this but I haven't managed to get it to work yet.